run/run_app_pm2

```diff
@@ -1,10 +1,8 @@
 #!/bin/bash
+pm2() {
+  yarn --cwd ./backend/ pm2 $1 $2 $3
+}
 
-if [ -z "$1" ]
-  then
-    echo "No config argument supplied"
-else
-    pm2 stop all
-    pm2 delete all
-    NODE_ENV=$1 NODE_CONFIG_DIR="./config" pm2 start "./run/3drepo.yml"
-fi
\ No newline at end of file
+pm2 stop all
+pm2 delete all
+pm2 start "../run/3drepo.yml"
```

Comment:
Does this needs changing? We lost the config details from the run script here.

Also it looks like you're not using this script anyway?

I think the original idea is that you should have pm2 installed globally. Servers def do (though I don't think servers rely on this script, it just runs 3drepo start/stop which outright calls pm2 i think @keifgwinn ?)

Reply:
3drepo start uses ./run_app_pm2 and these changes would break the multi-core useage on the servers and also break the use of the config file

the servers have pm2 installed globally (currently, docker doesn't use pm2 yet)

Reply:
In which case the change either has to revive the config setting, or just revert back to the old script @sanmont3drepo